src/Payloads/LoggedMailPayload.php

```diff
@@ -76,6 +84,19 @@ protected static function getMailContent(string $loggedMail, IMessage $message):
         return substr($loggedMail, $startOfHtml) ?? '';
     }
 
+    protected static function getMailAttachments(IMessage $message): array
+    {
+        return collect($message->getAllAttachmentParts())
+            ->map(function (MimePart $attachmentPart) {
+                return [
+                    'filename' => $attachmentPart->getFilename(),
+                    'content_id' => $attachmentPart->getContentId(),
+                    'content_type' => $attachmentPart->getContentType(),
+                    'content' => base64_encode($attachmentPart->getContent()),
```

Comment:
Is it worth checking the size of the content before doing this? Ray can sometimes struggle a bit with large payloads in my experience.